src/main/java/org/opensearch/knn/index/mapper/MethodFieldMapper.java

```diff
@@ -119,10 +122,23 @@ private MethodFieldMapper(
             knnMethodConfigContext
         );
         try {
-            this.fieldType.putAttribute(
-                PARAMETERS,
-                XContentFactory.jsonBuilder().map(knnLibraryIndexingContext.getLibraryParameters()).toString()
-            );
+            Map<String, Object> libParams = knnLibraryIndexingContext.getLibraryParameters();
+
+            // If VectorDataType is Byte using Faiss engine then manipulate Index Description to use "SQ8_direct_signed" scalar quantizer
```

Comment:
Can we do this in the FaissHNSWMethod. See here: https://github.com/opensearch-project/k-NN/blob/main/src/main/java/org/opensearch/knn/index/engine/faiss/AbstractFaissMethod.java#L99-L103